core/src/test/java/com/opensymphony/xwork2/ognl/OgnlValueStackTest.java

```diff
@@ -980,6 +1001,272 @@ public void testWarnAboutInvalidProperties() {
+        assertNull("able to access static method (result not null) ?", accessedValue);
+        accessedValue = ognlValueStack.findValue("@com.opensymphony.xwork2.ognl.OgnlValueStackTest@STATIC_FINAL_PUBLIC_ATTRIBUTE");
+        assertEquals("accessed static final public field value not equal to actual?", accessedValue, STATIC_FINAL_PUBLIC_ATTRIBUTE);
+        accessedValue = ognlValueStack.findValue("@com.opensymphony.xwork2.ognl.OgnlValueStackTest@STATIC_PUBLIC_ATTRIBUTE");
+        assertEquals("accessed static public field value not equal to actual?", accessedValue, STATIC_PUBLIC_ATTRIBUTE);
+        accessedValue = ognlValueStack.findValue("@com.opensymphony.xwork2.ognl.OgnlValueStackTest@STATIC_FINAL_PACKAGE_ATTRIBUTE");
+        assertNull("accessed final package field (result not null) ?", accessedValue);
+        accessedValue = ognlValueStack.findValue("@com.opensymphony.xwork2.ognl.OgnlValueStackTest@STATIC_PACKAGE_ATTRIBUTE");
+        assertNull("accessed package field (result not null) ?", accessedValue);
+        accessedValue = ognlValueStack.findValue("@com.opensymphony.xwork2.ognl.OgnlValueStackTest@STATIC_FINAL_PROTECTED_ATTRIBUTE");
+        assertNull("accessed final protected field (result not null) ?", accessedValue);
+        accessedValue = ognlValueStack.findValue("@com.opensymphony.xwork2.ognl.OgnlValueStackTest@STATIC_PROTECTED_ATTRIBUTE");
+        assertNull("accessed protected field (result not null) ?", accessedValue);
+        accessedValue = ognlValueStack.findValue("@com.opensymphony.xwork2.ognl.OgnlValueStackTest@STATIC_FINAL_PRIVATE_ATTRIBUTE");
+        assertNull("accessed final private field (result not null) ?", accessedValue);
+        accessedValue = ognlValueStack.findValue("@com.opensymphony.xwork2.ognl.OgnlValueStackTest@STATIC_PRIVATE_ATTRIBUTE");
+        assertNull("accessed private field (result not null) ?", accessedValue);
+    }
+
+    /**
+     * Test a default OgnlValueStackFactory and OgnlValueStack generated by it
+     * when no static access flags are set (not present in configuration).
+     */
+    public void testOgnlValueStackFromOgnlValueStackFactoryNoFlagsSet() {
+        OgnlValueStackFactory ognlValueStackFactory = new OgnlValueStackFactory();
+        DummyTextProvider dummyTextProvider = new DummyTextProvider();
+        XWorkConverter xWorkConverter = container.getInstance(XWorkConverter.class);
+
+        try {
+            reloadTestContainerConfigurationNoStaticAccessFlagsSet();
+            ognlValueStackFactory.setContainer(container);
+            ognlValueStackFactory.setTextProvider(dummyTextProvider);
+            ognlValueStackFactory.setXWorkConverter(xWorkConverter);
+        }
+        catch (Exception ex) {
+            fail("Unable to reload container configuration and set valuestackfactory - exception: " + ex);
+        }
+
+        OgnlValueStack ognlValueStack = (OgnlValueStack) ognlValueStackFactory.createValueStack();
```

Comment:
Same here. Additionally I feel above object creations (`new ...`s) and `set...`s shouldn't be needed. Calling `reload...` then `OgnlValueStackTest.createValueStack` should be enough.